apps/web/src/app/auth/trial-initiation/content/enterprise-content.component.html

```diff
@@ -1,17 +1,34 @@
-<h1 class="tw-text-4xl !tw-text-alt2">The Bitwarden Password Manager</h1>
+<h1 class="tw-text-3xl !tw-text-alt2">The Password Manager Trusted by Millions</h1>
 <div class="tw-pt-32">
-  <h2 class="tw-text-2xl">
-    Trusted by millions of individuals, teams, and organizations worldwide for secure password
-    storage and sharing.
-  </h2>
+  <h2 class="tw-text-2xl">Everything enterprises need out of a password manager:</h2>
 </div>
-<ul class="tw-mt-12 tw-flex tw-flex-col tw-gap-10 tw-text-2xl tw-text-main">
-  <li>Store logins, secure notes, and more</li>
-  <li>Collaborate and share securely</li>
-  <li>Access anywhere on any device</li>
-  <li>Create your account to get started</li>
+<ul class="tw-mt-12 tw-flex tw-flex-col tw-gap-10 tw-text-2xl tw-text-main tw-list-none tw-pl-0">
+  <li><i class="bwi bwi-lg bwi-check-circle tw-mr-4"></i>Secure password sharing</li>
+  <li>
+    <i class="bwi bwi-lg bwi-check-circle tw-mr-4"></i>Easy, flexible SSO and SCIM integrations
+  </li>
+  <li><i class="bwi bwi-lg bwi-check-circle tw-mr-4"></i>Free families plan for users</li>
+  <li><i class="bwi bwi-lg bwi-check-circle tw-mr-4"></i>Quick import and migration tools</li>
+  <li><i class="bwi bwi-lg bwi-check-circle tw-mr-4"></i>Simple, streamlined user experience</li>
+  <li><i class="bwi bwi-lg bwi-check-circle tw-mr-4"></i>Priority support and trainers</li>
 </ul>
 <div class="tw-mt-28 tw-flex tw-flex-col tw-items-center tw-gap-5">
-  <app-logo-forbes></app-logo-forbes>
-  <app-logo-us-news></app-logo-us-news>
+  <app-logo-cnet-5-stars></app-logo-cnet-5-stars>
+  <div class="tw-flex tw-items-end tw-gap-8">
+    <review-logo
+      [logoClass]="'tw-w-8'"
+      [logoSrc]="'../../images/register-layout/g2-logo.svg'"
+      [logoAlt]="'G2 Logo'"
+    ></review-logo>
```

Comment:
Since you aren't binding to any value in the `enterprise.component.ts` class file, you can just set these to a string without the `[]`.

```suggestion
    <review-logo
      logoClass="tw-w-8"
      logoSrc="../../images/register-layout/g2-logo.svg"
      logoAlt="G2 Logo"
    ></review-logo>
```